app/src/main/java/com/example/mycalculator/mvp/CalculatorPresenterImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.example.mycalculator.mvp;
+
+public class CalculatorPresenterImpl implements CalculatorContract.CalculatorPresenterContract {
+
+    private CalculatorContract.CalculatorModelContract model;
+    private CalculatorContract.CalculatorViewContract view;
+
+    public CalculatorPresenterImpl(CalculatorModelImpl model, CalculatorViewImpl view) {
```

Comment:
here maybe you can specify the contract type in the paremeters

Reply:
Done. Thx!